common/v2/features/Dashboard/constants.ts

```diff
@@ -7,7 +7,27 @@ import buyIcon from 'common/assets/images/icn-buy.svg';
 import swapIcon from 'common/assets/images/icn-swap.svg';
 import sendIcon from 'common/assets/images/icn-send.svg';
 import receiveIcon from 'common/assets/images/icn-receive.svg';
-import hardwareWalletIcon from 'common/assets/images/icn-hardware-wallet.svg';
+import ledgerIcon from 'common/assets/images/wallets/ledger.svg';
+import trezorIcon from 'common/assets/images/wallets/trezor.svg';
+
+const randomWallet = () => {
+  const hardwareWallets = [
+    {
+      icon: ledgerIcon,
+      title: translateRaw('DASHBOARD_ACTIONS_GET_WALLET_TITLE'),
+      link: EXT_URLS.LEDGER_REFERRAL.path,
+      description: translateRaw('DASHBOARD_ACTIONS_GET_WALLET_SUBTITLE', { $wallet: 'Ledger' })
+    },
+    {
+      icon: trezorIcon,
```

Comment:
These icons are very dark compared to the other CTA images, can we possibly make them a bit opaque? Maybe `opacity: 0.8;` as we do in the `/add-account` flow?

Reply:
Yes we can !